aspnetcore/grpc/performance.md

```diff
@@ -72,6 +72,29 @@ There are a couple of workarounds for .NET Core 3.1 apps:
 > * Thread contention between streams trying to write to the connection.
 > * Connection packet loss causes all calls to be blocked at the TCP layer.
 
+## Enable `ServerGarbageCollection` in high load client apps
+
+The .NET garbage collector has two options: workstation garbage collection (GC) and server garbage collection. Each is each tuned for different workloads. ASP.NET Core apps use server GC by default.
+
+Highly concurrent apps generally perform better with server GC. If a gRPC client app is sending and receiving a high number of gRPC calls at the same time then there may be a performance benefit in updating the app to use server GC.
+
+To enable server GC, set `<ServerGarbageCollection>` in the app's project file:
+
+```xml
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <ServerGarbageCollection>true</ServerGarbageCollection>
+  </PropertyGroup>
+
+</Project>
```

Comment:
If the code only has the relevant bit, the built-in article code **Copy** button is more useful ...

```suggestion
<PropertyGroup>
  <ServerGarbageCollection>true</ServerGarbageCollection>
</PropertyGroup>
```